demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -63,9 +63,8 @@ def are_release_notes_complete(self):
                     if file_name and file_type and file_type in RN_HEADER_BY_FILE_TYPE:
                         if (RN_HEADER_BY_FILE_TYPE[file_type] not in self.latest_release_notes) or \
                                 (file_name not in self.latest_release_notes):
-                            entity_name = update_rn_util.get_display_name(file)
```

Comment:
How come this is not longer needed?

Reply:
It was unnecessary before as we already got the same value from the same place (get_display_name) in the `file_name` variable (line 62).